doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
We would scrutinize the submitted patch so much that they would end up manually editing the changed lines by hand. :stuck_out_tongue: